package.json

```diff
@@ -75,7 +75,6 @@
     "lint-staged": "^13.1.0",
     "markdown-table": "^3.0.3",
     "open-cli": "^7.1.0",
-    "prettier": "^2.8.0",
```

Comment:
Would this break Prettier resolution by the vscode extension?

From https://github.com/prettier/prettier-vscode/blob/160b0e92d88fa19003dce2745d5ab8c67e886a04/README.md#prettier-resolution:

> This extension will use prettier from your project's local dependencies (recommended)

We may need to update the vscode workspace config to include `prettier.prettierPath` to help the vscode extension. From https://github.com/prettier/prettier-vscode/blob/160b0e92d88fa19003dce2745d5ab8c67e886a04/README.md#prettierprettierpath

> #### prettier.prettierPath
> 
> Supply a custom path to the prettier module. This path should be to the module folder, not the bin/script path. i.e. `./node_modules/prettier`, not `./bin/prettier`.

Reply:
I didn't know that this was going to influence vscode, I'm going to return it to its initial state.